app/src/main/java/com/babylon/wallet/android/data/gateway/serialisers/AccountLockerVaultCollectionItemSerializer.kt

```diff
@@ -19,7 +19,7 @@ object AccountLockerVaultCollectionItemSerializer : JsonContentPolymorphicSerial
         return when (element.jsonObject["type"]?.jsonPrimitive?.content) {
             AccountLockerVaultCollectionItemType.Fungible.value -> AccountLockerVaultCollectionItemFungible.serializer()
             AccountLockerVaultCollectionItemType.NonFungible.value -> AccountLockerVaultCollectionItemNonFungible.serializer()
-            else -> error("")
+            else -> error("AccountLockerVaultCollectionItemType not found.")
```

Comment:
Maybe it would be better to do something like this 
```
when (val type = element.jsonObject["type"]?.jsonPrimitive?.content) {
     ....
    else -> error("$type is not supported as an AccountLockerVaultCollectionItemType")
}

Reply:
right much better!